readme.md

```diff
@@ -1,8 +1,8 @@
 # Hello world
 
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_hello-world/)
```

Comment:
don`t use this task

Reply:
- [DEMO LINK](https://mrogozhinskaya.github.io/layout_hello-world/)
- [TEST REPORT LINK](https://mrogozhinskaya.github.io/layout_hello-world/report/html_report/)